pages/apis/graphql/cookbooks/organizations.md

```diff
@@ -239,7 +239,30 @@ mutation CreateUser {
 }
 ```
 
+## Get an Organization Member Creation Timestamp
```

Comment:
Thanks for this contribution @stephanieatte ! 🎉 

As per our [writing styleguide](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#headings), please convert this heading to sentence case to allow the docs builds to pass the linting process.